end-to-end-tests/features/violations/message_too_long.feature

```diff
@@ -0,0 +1,23 @@
+Feature: When the commit title exceeds the configured maximum length, it is picked up as a linting violation.
+
+
+  Scenario: Long message fails with default limit then passes with increased limit
+    Given the context and environment are reset.
+    When linting the "feat: this is a very long commit message that exceeds fifty characters".
```

Comment:
Pull the commit message out to a table like  the preceding whitespace violation feature file.